doc/source/cluster/running-applications/monitoring-and-observability.rst

```diff
@@ -188,3 +188,14 @@ Ray Cluster information. Here, we will use a Python script and the
                     'object_store_memory': 2.0},
      'alive': True}]
     """
+
+Setting up Grafana and Metrics
+##############################
+
+One common pattern of supporting :ref:`metrics <ray-metrics>` on a remote ray cluster is by running the Prometheus and Grafana services
```

Comment:
Can you make this as the top section instead of `Prometheus metrics`? 

Setting up Grafana and Metrics
^^^^^^^^^^^^^^^^^^^^^^^^

One common pattern ...

Prometheus metrics
###############